components/org.wso2.carbon.identity.captcha/pom.xml

```diff
@@ -174,10 +174,10 @@
                             org.apache.commons.lang; version="${commons-lang.wso2.osgi.version.range}",
                             org.apache.commons.collections; version="${commons-collections.wso2.osgi.version.range}",
                             org.apache.commons.logging.*; version="${commons-logging.osgi.version.range}",
-                            org.apache.http.*,
-                            org.apache.http.client.*;version="${httpcomponents-httpclient.imp.pkg.version.range}",
-                            org.apache.http.impl.client.*;version="${httpcomponents-httpclient.imp.pkg.version.range}",
-                            org.apache.http.message.*;version="${httpcore.osgi.version.range}",
+                            org.apache.hc.*,
+                            org.apache.hc.client5.http.*;version="${httpcomponents-httpclient.imp.pkg.version.range}",
+                            org.apache.hc.client5.http.impl.*;version="${httpcomponents-httpclient.imp.pkg.version.range}",
+                            org.apache.hc.core5.http.message.*;version="${httpcore.osgi.version.range}",
```

Comment:
We might need to import org.wso2.carbon.utils.httpclient5.HTTPClientUtils as well